test_util/main.tf

```diff
@@ -56,7 +62,7 @@ module "dcos" {
   }
 
   cluster_name        = "${local.cluster_name}"
-  ssh_public_key_file = "~/.ssh/id_rsa.pub"
+  ssh_public_key_file = "./tf-dcos-rsa.pem.pub"
```

Comment:
Please do not introduce hardcoded values since it breaks workflow in other parts. I moved it to variable https://github.com/dcos/dcos/pull/6974